pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -49,7 +49,7 @@ const (
 	fsidSecretNameKey = "fsid"
 	MonSecretNameKey  = "mon-secret"
 	// AdminSecretName is the name of the admin secret
-	adminSecretNameKey = "admin-secret"
+	AdminSecretNameKey = "admin-secret"
```

Comment:
why is the capatilizing is required? Previously it was working fine

Reply:
It's a cross package import, so I believe this is a must.